tests/example_stdin_data

```diff
@@ -0,0 +1 @@
+This data is passed to the command via stdin
```

Comment:
Minor: "stdin" is usually input from the terminal. While this is input from a file. This is a bit confusing when reading the test below. Can we maybe change this to
```suggestion
This text is part of a file
```